rolling/distribution.yaml

```diff
@@ -3094,6 +3094,22 @@ repositories:
       url: https://github.com/ros-sports/nao_lola.git
       version: rolling
     status: developed
+  naoqi_libqi:
+    doc:
+      type: git
+      url: https://github.com/ros-naoqi/libqi.git
+      version: ros2
+    release:
+      tags:
+        release: release/rolling/{package}/{version}
+      url: https://github.com/ros-naoqi/libqi-release.git
```

Comment:
To go into Rolling, the release repository needs to be in the `ros2-gbp` organization.  See the documentation at https://github.com/ros2-gbp/ros2-gbp-github-org/blob/latest/CONTRIBUTING.md for how to get this release repository created.

Reply:
From the documentation I do not understand what action I should take to do that.
I think it does not matter to get a release for  ̀rolling`.
Did I miss some simple solution?
Otherwise I will drop this PR.

Reply:
> From the documentation I do not understand what action I should take to do that.

You would need to create an issue or PR to https://github.com/ros2-gbp/ros2-gbp-github-org with your release team and the repository you want to import.  Then the maintainer of that repository will create the release repository for you, at which point you would re-release using that release repository.  This is required for Rolling so that we can automatically branch the repository when we branch off of Rolling at release time.

If you don't want to do any of that now, you can just close this PR.  But do note that if you ever want this package in Rolling (and to get automatically released into the next ROS 2 distribution), you'll have to go through this process in the future.